enterprise/server/atime_updater/atime_updater.go

```diff
@@ -116,6 +118,25 @@ func (u *atimeUpdater) groupID(ctx context.Context) string {
 	return user.GetGroupID()
 }
 
+func (u *atimeUpdates) findPendingUpdate(instanceName string, digestFunction repb.DigestFunction_Value) *atimeUpdate {
```

Comment:
nit: since this does a mutation, the naming could be a bit clearer.

```suggestion
func (u *atimeUpdates) findOrCreatePendingUpdate(instanceName string, digestFunction repb.DigestFunction_Value) *atimeUpdate {
```

(Unrelated to the PR but I had some trouble grokking this because it took me a minute to realize that `atimeUpdate` is actually a batch of atime updates)